api/documents/schema.graphql

```diff
@@ -17,7 +17,25 @@ extend type User {
   documents(
     "Filtering options for documents returned."
     filterBy: DocumentFilters
-  ): [Document!]!
+    "Number of documents to be returned"
+    first: Int
+    "Cursor for pagination"
+    after: String
+  ): UserDocumentsConnection!
+}
+
+type UserDocumentsConnection {
```

Comment:
Can you please also add documentation for the fields of the newly added types (can mostly be copied from eg https://docs.github.com/en/graphql/reference/objects#pageinfo)